controllers/hypershiftlogforwarder/hypershiftlogforwarder_controller.go

```diff
@@ -120,23 +118,55 @@ func (r *HyperShiftLogForwarderReconciler) Reconcile(ctx context.Context, req ct
 		return ctrl.Result{}, err
 	}
 
-	deletion := false
-	// HyperShiftLogForwarder was deleted by CU
-	if !instance.DeletionTimestamp.IsZero() {
-		deletion = true
+	if !instance.ObjectMeta.DeletionTimestamp.IsZero() {
```

Comment:
I am not quite sure what is the difference between the two, I see both of them are used in different repos

Reply:
According to [this](https://kubebuilder.io/reference/using-finalizers.html) article `ObjectMeta.DeletionTimestamp` may be right way